docs/vsphere.md

```diff
@@ -0,0 +1,13 @@
+# VMWware vsphere
+
+To use the machine-controller to create machines on VMWare vsphere, you must first
+create a template:
+
+1. Go into the VSphere WebUI, select your datacenter, right click onto it and choose "Deploy OVF Template"
+2. Enter `https://cloud-images.ubuntu.com/releases/16.04/release/ubuntu-16.04-server-cloudimg-amd64.ova` into the
```

Comment:
Please add a line for centos and coreos

Reply:
Those are currently not tested on vsphere, I will add docs once that's done (probably not within this PR)